security/pfSense-pkg-snort/files/usr/local/www/snort/snort_alerts.php

```diff
@@ -1084,7 +1087,7 @@ function snort_match_filter_field($flent, $fields, $exact_match = FALSE) {
 				<td><?=$alert_src_p; ?></td>
 				<td style="word-wrap:break-word; white-space:normal"><?=$alert_ip_dst;?></td>
 				<td><?=$alert_dst_p; ?></td>
-				<td><?=$alert_sid_str; ?><br/><?=$sidsupplink; ?>&nbsp;&nbsp;<?=$sid_dsbl_link; ?>&nbsp;&nbsp;<?=$sid_action_link; ?></td>
+				<td><a target="_blank" href=<?=$link_sid_str; ?>><?=$alert_sid_str; ?></a><br/><?=$sidsupplink; ?>&nbsp;&nbsp;<?=$sid_dsbl_link; ?>&nbsp;&nbsp;<?=$sid_action_link; ?></td>
```

Comment:
`href` target should be surrounded by double quotes, so `<a target="_blank" href="<?=$link_sid_str?>">`

Reply:
Fixed